app/assets/javascripts/modules/namespaces/components/table.js

```diff
@@ -14,6 +14,12 @@ export default {
     namespaces: {
       type: Array,
     },
+    namespacesUrl: {
+      type: String,
+    },
+    webhooksPath: {
```

Comment:
I think we should follow the same rule here: either both end with `Url` or with `Path`.

Reply:
The idea with `url` was to have the full url included (e.g.: http://dev/namespaces) and for `path` is just the path itself (e.g.: /namespaces). But I think we can use only path for namespaces. Will change that.